contribute_documentation.rst

```diff
@@ -139,10 +139,16 @@ following command ::
 
 HTML files are generated in _build/html directory. From a separate terminal
 window, you can have a simple Python webserver always serving up this content
-with the following command ::
+with one of the following commands ::
```

Comment:
```suggestion
with one of the following commands:
```

Reply:
@jonpwilson So had another look and it seems at least my version of Sphinx is worried about this bit.
By leaving this prior literal block indicator in place it throws the "Literal block expected; none found." error so I'm suggesting we pop in a straight colon here.